src/main/java/org/example/javaproglovo/service/ItemService.java

```diff
@@ -0,0 +1,33 @@
+package org.example.javaproglovo.service;
+
+import lombok.AllArgsConstructor;
+import org.example.javaproglovo.converter.ItemConverter;
+import org.example.javaproglovo.dto.ItemDto;
+import org.example.javaproglovo.repository.ItemRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@AllArgsConstructor
+@Service
+public class ItemService {
+
+    private ItemRepository itemRepository;
```

Comment:
according to the task, objects should be stored in memory, meaning in Map or something